kaggle_cli/submit.py

```diff
@@ -25,7 +25,7 @@ def get_parser(self, prog_name):
         parser.add_argument('-u', '--username', help='username')
         parser.add_argument('-p', '--password', help='password')
         parser.add_argument('-z', '--zip', type=self._str2bool, nargs='?', const=True, default=False,
```

Comment:
Why don't you use the same argument setting as `config.py`?

i.e. `parser.add_argument('-z', '--zip', help='zip the submission file before uploading?', action='store_true')`

Reply:
agree